### DISCUSSION TITLE
Wasabi S3

### DISCUSSION THREAD

**Discussion by GSmithRE** [fluent/fluentd#4276] [Q&A]
Hello

Does anyone use fluentd with a Wasabi S3 Bucket, perhaps just a few teaks to the AWEs plugin?

Thanks
Giles.

**Comment by kenhys** [comment]
It seems that s3 compatible plugin will work.
It may be careful about signature version.

https://github.com/fluent/fluent-plugin-s3/blob/master/docs/output.md#signature_version
asically s3
